docker-compose.yml

```diff
@@ -28,9 +28,7 @@ services:
     volumes:
       - ${MONDEY_SSL_CERT:-./cert.pem}:/mondey_ssl_cert.pem
       - ${MONDEY_SSL_KEY:-./key.pem}:/mondey_ssl_key.pem
-#  email:
-#    image: "boky/postfix"
-#    environment:
-#      - ALLOW_EMPTY_SENDER_DOMAINS="true"
-#    networks:
-#      - mondey-network
+  email:
+    image: "boky/postfix"
+    environment:
+      - ALLOW_EMPTY_SENDER_DOMAINS="true"
```

Comment:
I looked this up [here]( https://github.com/bokysan/docker-postfix/blob/master/README.md#allowed_sender_domains-and-allow_empty_sender_domains) ) but could only find documentation in conjunction with `ALLOWED_SENDER_DOMAINS`. Is this temporary?

Reply:
`ALLOWED_SENDER_DOMAINS` is I think by default empty, so setting `ALLOW_EMPTY_SENDER_DOMAINS` tells postfix to allow sending emails from any domain